modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/objects/AffectedEntryResult.java

```diff
@@ -37,71 +37,71 @@
 /**
  * Result with lists of broken and fixed, skipped entries.
  */
-public class PartitionReconciliationResult extends IgniteDataTransferObject {
+public class AffectedEntryResult extends IgniteDataTransferObject {
```

Comment:
Maybe AffectedEntriesResult or ReconciliationAffectedEntries would be better?
Looks like this class encloses just one entry.